students/seungri/user/views.py

```diff
@@ -36,3 +37,29 @@ def post(self, request):
         except IntegrityError:
             return JsonResponse({"message": "DUPLICATE_ENTRY_KEY_ERROR"}, status=400)
 
+class UserLogIn(View):
+    def post(self, request):
+        try:
+            data            = json.loads(request.body)
+            email           = data['email']
+            input_password  = data['password'].encode('utf-8')
+
+            if email == "" or input_password == "":
+                return JsonResponse({"message": "KEY_ERROR"}, status=400)
+
+            login_user  = User.objects.get(email=email)
+            db_password = login_user.password
+            db_password = db_password.encode('utf-8')
```

Comment:
이정도는 합쳐주시는게 더 깔끔할 것 같네요.
```suggestion
            db_password = login_user.password.encode('utf-8')
```

Reply:
수정했습니다!